services.html

```diff
@@ -0,0 +1,131 @@
+
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea! - Services</title>
+  <link rel="icon" type="image/png" href="/img/header-img.png">
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=-1">
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+    <div class="container">
+     <header>
+         <nav class="navigation">
+            <a href="index.html"> <img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+            <a class="navi" href="services.html">Services</a>
+            <a class="navi" href="">Product</a>
+            <a class="navi" href="">Vision</a>
+            <a class="navi" href="">Features</a>
+            <a class="navi" href="">About</a>
+            <a class="navi" href="">Contact</a>
+        </nav>
+        </header>
+      <img class="services-header-img" src="img/services-header.jpg" alt="services image">
+
+      <section class="middle">
+          <h1>Services</h1>
+          <p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
+      </section>
+
+      <div class="break"></div>
+
+      <section class="row-1">
+          <div class="card-left">
+              <h2>Digital Design</h2>
+              <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+              eleifend. Mauris euismod facilisis iaculis.</p>
+              <button>
+                  Learn More
+              </button>
+          </div>
+
+          <div class="card-right">
+              <h2>UX / UI</h2>
+              <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+              eleifend. Mauris euismod facilisis iaculis.</p>
+
+              <button>
+                  Learn More
+              </button>>
+          </div>
+      </section>
+
+
+      <section class="row-2">
+            <div class="card-left">
```

Comment:
where your styling theses classes I would copy the classes and add services to the front of the class. So that you may edit them separately and style them to your desire.